base_layer/core/src/base_node/sync/rpc/tests.rs

```diff
@@ -75,7 +75,7 @@ mod sync_blocks {
     }
 
     #[tokio::test]
-    async fn it_sends_an_empty_response() {
+    async fn it_does_not_send_empty_responses_with_bad_requests() {
```

Comment:
```suggestion
    async fn is_sends_bad_request+on_bad_response() {
```
In a year we wont know that we sent and empty response and we will wonder why this test exists and what it tests